velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -286,6 +286,106 @@ TEST_F(VeloxSubstraitRoundTripTest, ifThen) {
   assertPlanConversion(plan, "SELECT if (c0=1, c0 + 1, c1 + 2) as x FROM tmp");
 }
 
+TEST_F(VeloxSubstraitRoundTripTest, orderBySingleKey) {
+  auto vectors = makeVectors(3, 4, 2);
+  createDuckDbTable(vectors);
+  auto plan = PlanBuilder()
+                  .values(vectors)
+                  .orderBy({"c0 DESC NULLS LAST"}, false)
```

Comment:
Should we test other options, i.e. ASC NULLS FIRST?

Reply:
using other options in the `orderBy` and `orderByPartial` tests.